Sources/ApiVideoPlayer/Views/SpeedometerListView.swift

```diff
@@ -0,0 +1,104 @@
+#if !os(macOS)
```

Comment:
This could be factorized with `SubtitleListView`.

1 - Create a generic class `SelectableListView<T>` that takes a list of value and a value as input.
2 - You can create specific list view like this:
```
let speedometerListView = SelectableListView<Float>(frame, values = [0.5, 1.0, 1.25, 1.5, 2.0], selectedValue = 1.0)
```